CodePush.ios.js

```diff
@@ -126,72 +126,106 @@ function checkForUpdate() {
  * releases, and displaying a standard confirmation UI to the end-user
  * when an update is available.
  */
-function sync(options = {}) {  
+function sync(options = {}, syncStatusChangeCallback, downloadProgressCallback) {  
   var syncOptions = {
-    descriptionPrefix: " Description: ",
-    appendReleaseDescription: false,
     
     ignoreFailedUpdates: true,
-    
-    mandatoryContinueButtonLabel: "Continue",
```

Comment:
They are, the line `syncOptions.updateDialog = Object.assign(CodePush.DEFAULT_UPDATE_DIALOG, syncOptions.updateDialog);` handles that.